edge/pkg/metamanager/process.go

```diff
@@ -508,10 +508,10 @@ func (m *metaManager) processNodeConnection(message model.Message) {
 }
 
 func (m *metaManager) processSync(message model.Message) {
-	m.syncPodStatus()
+	m.syncPodStatus(message)
```

Comment:
The `message` is no need here?

Reply:
The message is just used to tell the edge to sync podstates with its operation，it's dispensable in func processSync , I add it for UT that it will send err message to cloud when unmarshal podstatus failed.

Reply:
Actually the `feedbackError` will return the message to edged, because it will judge the message source, the source is edged, right? Btw, I think we can delete the message parameter directly here.